Android/ConstraintLayout/samples/.vs/ConstraintLayoutSample/xs/UserPrefs.xml

```diff
@@ -0,0 +1,10 @@
+﻿<Properties StartupConfiguration="{17133E29-65C3-4B55-AC7E-107F50EE21D6}|Default">
```

Comment:
this file should be removed

Reply:
In my PR I added a .vs/ to .gitignore, so you can do the same and see who gets merged first :)